src/main/java/fr/esgi/al5_2/Tayarim/services/ProprietaireService.java

```diff
@@ -42,12 +43,18 @@ public ProprietaireDTO creerProprietaire(@NonNull ProprietaireCreationDTO propri
         }
         proprietaireCreationDTO.setNumTel(numTel);
 
-        String hashedPassword = hashPassword(proprietaireCreationDTO.getMotDePasse());
 
-        proprietaireCreationDTO.setMotDePasse(hashedPassword);
+        StringBuilder generatedPassword = new StringBuilder(16);
+        String allowedchar = "abcdefghijklmnopqrstuvwxyz0123456789!@#$%&*()_+-=[]?";
+        SecureRandom random = new SecureRandom();
+        for (int i = 0; i < 16; i++) {
+            int index = random.nextInt(allowedchar.length());
+            generatedPassword.append(allowedchar.charAt(index));
+        }
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'for rcurly' has incorrect indentation level 8, expected level should be 4.